vitest.nuxt.config.ts

```diff
@@ -13,6 +13,7 @@ export default defineVitestConfig({
     environmentOptions: {
       nuxt: {
         overrides: {
+          buildId: 'nuxt-app',
```

Comment:
**Note to self**: before release we need to update `@nuxt/test-utils` to set `buildId` to `nuxt-app` in the vitest environment...